packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -112,7 +112,7 @@ export default class Calendar extends React.Component {
      * @private
      * Internationalization object with translation APIs. Provided by `injectIntl`.
      */
-    intl: intlShape,
+    intl: PropTypes.shape({}),
```

Comment:
Why are these shapes not as defined as the others?

Reply:
It's because this component doesn't actually use the `intl` object, it just passes it through as a prop. I had only defined the shape with what each component used from the `intl` object in lieu of defining the whole shape everywhere . If I dig down to where it's actually used, it looks like it's being used for the `formatMessage` function,  I can go ahead and define that here.